mne/viz/utils.py

```diff
@@ -1507,6 +1507,7 @@ def _plot_sensors(pos, colors, bads, ch_names, title, show_names, ax, show,
                          ch_names=ch_names, show_names=show_names)
         fig.canvas.mpl_connect('pick_event', picker)
 
+    ax.axis("off")  # remove border around figure
```

Comment:
IMO the grid is good to have for the 3d plot. You can easily see if the coordinate frame is correct or not.